sample_apps/simple_app_under_oe/test.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+# ##############################################################################
+# test.sh: Execute build-and-test of simple_app_under_oe with diff arguments
```

Comment:
I added this driver shell script so that once all these changes stabilize, one can simply run this manually on an sgx-enabled box, to exercise all interfaces in one single script.